### DISCUSSION TITLE
DNS based network policies

### DISCUSSION THREAD

**Discussion by DenisBiondic** [Azure/AKS#2752] [General]
What are the officially supported options to implement DNS based network policies in AKS?

**Comment by maur1** [comment]
Hi,

I am currently using the Calico as a Network policy engine. Allows me to block traffic on L7 level. The calico plugin is stated to be supported by azure and calico support [here](https://learn.microsoft.com/en-us/azure/aks/use-network-policies#differences-between-network-policy-engines-cilium-azure-npm-and-calico). Although only using it as a policy engine, it is quite a disruptive tool and is hard to uninstall. So would be happy to see the same functionality be delivered by the AKS team itself.
